code/modules/cargo/packs/science.dm

```diff
@@ -41,6 +81,33 @@
 					/obj/item/integrated_electronics/wirer)
 	crate_name = "circuitry starter pack crate"
 
+/datum/supply_pack/science/monkey
+	name = "Monkey Cube Crate"
+	desc = "Stop monkeying around! Contains seven monkey cubes. Just add water!"
+	cost = 2000
+	contains = list (/obj/item/storage/box/monkeycubes)
+	crate_name = "monkey cube crate"
+
+/datum/supply_pack/science/nitrilegloves
+	name = "Nitrile Gloves Crate"
+	desc = "Ever handling toxic chemicals? Well worry not to keep your flesh with some nitrile made gloves! Contains three pares of nitrile gloves."
+	cost = 1500
+	contains = list(/obj/item/clothing/gloves/color/latex/nitrile,
+					/obj/item/clothing/gloves/color/latex/nitrile,
+					/obj/item/clothing/gloves/color/latex/nitrile)
+	crate_name = "nitrile gloves crate"
+
+/datum/supply_pack/science/nuke_b_gone
```

Comment:
Unsure how safe it is making traitor tools available to cargo, the same department that jumps on gun crates when a cricket chirps, but I’ll leave that to Kevin/whoever looks at this.

Reply:
Idea is that traitors are not buying this one.